roles/openshift_certificate_expiry/test/test_fakeopensslclasses.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+'''
+ Unit tests for the FakeOpenSSL classes
+'''
+# Run from roles/openshift_certificate_expiry/
+#     $ nosetests -v
+
+import os
+import sys
+import unittest
+
+# Removing invalid variable names for tests so that I can
+# keep them brief
+# pylint: disable=invalid-name,no-name-in-module
+# Disable import-error b/c our libraries aren't loaded in jenkins
+# pylint: disable=import-error,wrong-import-position
+# place class in our python path
+module_path = os.path.realpath('./library/')
```

Comment:
Tried this. Still failed. We talked on IRC and decided to use the `pytest` skip functionality for now. The test is still part of the source tree, but it is not ran at this time.